test/lsp-clangd-test.el

```diff
@@ -24,6 +24,24 @@
 (require 'ert)
 (require 'lsp-clangd)
 
+;; TODO move to a common/shared test-utils.el
```

Comment:
Isn't this in `lsp-integration-test` already? Can't you just `require` or `load` it?

Reply:
Yes it is - is it ok, if I move these helpers out to a `test-utils.el` file? I am not knowledgeable enough to know about macro evaluation if I decide to load lsp-integration-test.el in lsp-clangd-test.el

Reply:
> I am not knowledgeable enough to know about macro evaluation if I decide to load lsp-integration-test.el in lsp-clangd-test.el

That should be fine.